packages/applications/types/file-folders.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface FolderDetails {
```

Comment:
I didn't know backend had a secret folder for types

(can resolve this, it's me being surprised.)